sky/clouds/gcp.py

```diff
@@ -347,6 +358,46 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
                 'For more info: '
                 'https://skypilot.readthedocs.io/en/latest/getting-started/installation.html'  # pylint: disable=line-too-long
             )
+
+        # Check APIs.
+        project_id = self.get_project_id()
+        apis = (
+            ('compute', 'Compute Engine'),
+            ('cloudresourcemanager', 'Cloud Resource Manager'),
+            ('iam', 'Identity and Access Management (IAM)'),
+        )
+        enabled_api = False
+        for endpoint, display_name in apis:
+            if is_api_disabled(endpoint, project_id):
+                # For 'compute': ~55-60 seconds for the first run. If already
+                # enabled, ~1s. Other API endpoints take ~1-5s to enable.
+                if endpoint == 'compute':
+                    suffix = ' (may take a minute)'
+                else:
+                    suffix = ''
+                print(f'\nEnabling {display_name} API{suffix}...')
```

Comment:
Should we say this doesn't incur any cost? as last time a Stanford user was concerning about it.